.github/workflows/test.yml

```diff
@@ -97,6 +102,7 @@ jobs:
         with:
           playbook: playbook.yml
           directory: test
+          configuration: ansible.cfg
```

Comment:
This looks invalid?

Reply:
I didn't understand what you mean, would you please provide further details

Reply:
ah i see local test is failing, am on it